frontend/src/app/main/ui/ds/helpers.cljs

```diff
@@ -11,4 +11,12 @@
 
 (def default
   (mf/object
-   {:uuid parse-uuid}))
+   {:uuid parse-uuid
+    :linear :linear
```

Comment:
Is better expose a function that converts a string to keyword, instead of all possible keywords.
That helpers namespace/module is intended to expose helpers for build instances of objects and not sample values as-is.